server/src/test/java/io/crate/integrationtests/AlterTableIntegrationTest.java

```diff
@@ -257,6 +276,29 @@ private void execute_statements_that_drop_and_add_column_with_the_same_name_agai
         execute("alter table t drop column o['oo']");
         execute("alter table t add column o['oo'] object AS(a text)");
 
+        // Only top-level column is shown, children are completely gone
+        execute("""
+            select attname, attnum, attisdropped
+            from pg_attribute
+            where attrelid = 't'::regclass AND attname != 'p'
+            order by attnum"""
+        );
+
+        // There is no entry with OID 6, drop sub-columns are completely removed from the mapping.
+        pgAttributeRows = """
+            _dropped_1| 1| false
+            b| 2| false
+            o| 3| false
+            o['a']| 4| false
+            o['_dropped_5']| 5| false
+            a| %d| false
```

Comment:
Will fix that, for dropped column we don't need to show FQN to avoid such weird output.